libs/ddhub-client-gateway-frontend/ui/core/src/components/Table/Search/Search.tsx

```diff
@@ -15,6 +15,7 @@ export interface SearchProps {
   setFilter?: (value: string) => void;
   onSearchInput?: (value: string) => void;
   debounceTime?: number;
+  tableRows?: any[];
```

Comment:
is it possible to set an interface here? Or it might be anything here?

Reply:
This tableRows is of generic type and can be anything.